charts/kyverno/templates/imagepullsecret.yaml

```diff
@@ -0,0 +1,13 @@
+{{ range $name, $secret := .Values.imagePullSecrets }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels: {{ include "kyverno.labels" $ | nindent 4 }}
+    app: kyverno
+  name: {{ $name }}
+  namespace: {{ template "kyverno.namespace" $ }}
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: {{ template "kyverno.imagePullSecret" (list $secret) }}
```

Comment:
Why create a list with a single element ?

Reply:
You can pass $secret directly to the template.

Reply:
Made that adjustment, passing the data directly

Reply:
👍